core/solr_index.py

```diff
@@ -26,7 +26,7 @@ def conn():
     return pysolr.Solr(settings.SOLR)
 
 def page_count():
-    return len(conn().search(q='type:page', fl='id'))
+    return conn().search(q='type:page', rows=0).hits
```

Comment:
oooh this is way better

Reply:
Thanks!  Had to fix that because of a separate problem, but it's actually a significant improvement for searches with huge result sets.